js/script.js

```diff
@@ -194,19 +196,35 @@ $(function () {
     var cookieNames = {
         nsfwCookie: "nsfwCookie",
         shouldAutoNextSlideCookie: "shouldAutoNextSlideCookie",
-        timeToNextSlideCookie: "timeToNextSlideCookie"
+        timeToNextSlideCookie: "timeToNextSlideCookie",
+        soundCookie: "soundCookie"
     };
 
     var setCookie = function (c_name, value) {
-        Cookies.set(c_name, value, { expires: rp.settings.cookieDays });
+        Cookies.set(c_name, value, {expires: rp.settings.cookieDays});
     };
 
-    
+
     var getCookie = function (c_name) {
         // undefined in case nothing found
         return Cookies.get(c_name);
     };
 
+    var updateSound = function () {
+        rp.settings.sound = $('#sound').is(':checked');
+        setCookie(cookieNames.soundCookie, rp.settings.sound);
+        var videoTags = document.getElementsByTagName('video');
+        if (videoTags.length === 1) {
+            videoTags[0].muted = !rp.settings.sound;
+        }
+        var audioTags = document.getElementsByTagName('audio');
```

Comment:
since for v.redd.it audio tag is separate from video tag, this needs to be checked as well